rfcs/formats/network/iin_method.md

```diff
@@ -0,0 +1,264 @@
+
+## IIN Method Syntax
+
+The IIN Method scheme MUST begin with the prefix `did:iin`, and is defined by the following ABNF based on the [DID Syntax](https://www.w3.org/TR/did-core/#did-syntax):
+
+```
+iin-did                = "did:iin" iin-name ":" iin-specific-id
+iin-name        = 1*idchar
+iin-specific-id = *( *idchar ":" ) 1*idchar
+idchar             = ALPHA / DIGIT / "." / "-" / "_" / pct-encoded
+pct-encoded        = "%" HEXDIG HEXDIG
+```
+### DID Types
+
+In an IIN there can be two types of identifiers. 
+
+- First, for individuals / organizations which may participate in a blockchain network, called *Participant Unit DID*. 
+
+- Second, for blockchain networks (formed by a group of individuals / organizations / other networks), called *Network DID*.
+
+A *Network DID* 's `iin-specific-id` *MUST* end with ".network".
+
+### Examples:
+
+```
+did:iin:ibmiin:fastshippingcompany
+did:iin:ibmiin:tradelens.network
+did:iin:openiin:bankorg
+```
+
+
+## IIN Method Operations
+
+### Create
+
+For creating a new DID and a DID Document in the IIN, the DID method of the IIN must authenticate the DID controller who is creating a DID. For a Network DID, this authentication must be based on a `verificationMethod` of type `GroupMultiSig`. For other DIDs (individual / organizations), the authentication can be based on any [`verificationMethod`](https://w3c.github.io/did-spec-registries/#verification-method-types).
+
+
+#### For Participant Unit DID
+
+The DID registration (create method) request must contain the DID document, as well as a signature that can be verified against the `authentication` verification method specified in the DID document. This request is sent by the [`IIN Agent`](./iin-agent.md) of the participant. The underlying protocol dictating how the DID registration request will be sent to the IIN depends on the implementation the IIN registry.
+
+Example DID registration request:
+```json
+{
+  "DIDDocument": {
+    "id": "did:exampleiin:org1",
+    "verificationMethod": [{
+      "id": "did:exampleiin:org1#key1",
+      "type": "Ed25519VerificationKey2020",
+      "controller": "did:exampleiin:org1",
+      "publicKeyMultibase": "zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
+    }],
+
+    "authentication": ["did:exampleiin:org1#key1"]
+  },
+  "signature": "..."
+}
+```
+
+A DID registration request, that is the DID create method of the IIN registry, must authenticate the request. This authentication is to be done by validating the signature in the request with the help of the authentication method present in the DID Document of the request.
```

Comment:
"that is the DID create method" --> "or the DID 'create' method" (use double quotes for 'create' perhaps)